1.4/Source/VanillaPsycastsExpanded/Paths/Archotechist/Ability_AggressiveHeatDump.cs

```diff
@@ -12,16 +12,21 @@
 
     public class Ability_AggressiveHeatDump : Ability
     {
+        public override float GetRadiusForPawn() =>
+            Mathf.Ceil(Mathf.Min(this.pawn.psychicEntropy.EntropyValue / 20f, 9f * base.GetRadiusForPawn()));
+
+        public override float GetPowerForPawn() => DamageDefOf.Flame.defaultDamage *
+                                                   this.pawn.psychicEntropy.EntropyValue * base.GetPowerForPawn();
```

Comment:
```suggestion
                                                   (this.pawn.psychicEntropy.EntropyValue / 100f) * base.GetPowerForPawn();
```
Original divided by 100, otherwise it’s way too big.

Reply:
> Original divided by 100, otherwise it’s way too big.

That's why it's 0.01f in the xml

Reply:
Ah I see.